pages/embed.tsx

```diff
@@ -65,6 +74,8 @@ export const getServerSideProps: GetServerSideProps<IPageProps> = async ({
   // 3. Parse config and RSS data into embed data.
   const data = parseEmbedData(config, rssData);
 
+  log.info({ res }, 'Embed Response');
```

Comment:
I'm seeing a `"statusCode":null` for this, followed by a logged (in color) non-json line.  Is that just a dev thing?

```
# logs 1 line of json
{"level":30,"time":1733761788173,"pid":53379,"hostname":"airbook","name":"pages/embed.tsx","res":{"statusCode":null,"headers":{"cache-control":"public, max-age=300"}},"msg":"Embed Response"}

# and then 1 line of plain text
 GET /e?uf=https://publicfeeds.net/f/380/feed-rss.xml 200 in 1154ms
```

Reply:
Also - what if you just combined the req/res into a single log.info?  Most logging frameworks tend to do that - log the entire req/res cycle in a single statement.

That way, we know which req/res were tied together.

Reply:
That GET line is dev server output, I think.

I split the req and res output up to avoid errors generated creating the response interrupting the output of the req.

Reply:
If you want to keep them split, we at least need some unique `request_id` that can tie the 2 together.

But it does introduce 1 more step in debugging anything... you end up querying for responses where statusCode >= 500, and _then_ running a 2nd query for the requests associated with those.

Reply:
Pushed up a change to log req and res in a single log message, also error logs will include the req.

Reply:
Couldn't find a good way to add a request ID that would consistently be added to all log messages, so just went with making sure each log message had the related objects to debug.